pkg/io/config_store.go

```diff
@@ -1,25 +1,26 @@
 package io
 
 import (
+	"io/ioutil"
+
 	"github.com/jenkins-x/jx/pkg/util"
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v2"
-	"io/ioutil"
 )
 
 // ConfigStore provides an interface for storing configs
 type ConfigStore interface {
 	// Write saves some secret data to the store
-	Write(secretName string, bytes []byte) error
+	Write(fileName string, bytes []byte) error
```

Comment:
This interface shouldn't have reference to storing secrets in files (which is why I used a more abstract name).